docs/release-notes/v0.39.0.md

```diff
@@ -0,0 +1,72 @@
+# Announcing Radius v0.39.0
+
+Today we're happy to announce the release of Radius v0.39.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### Dapr Building Block: Configuration Store
+https://github.com/radius-project/radius/pull/7906
+
+### Add an in-memory data store
+https://github.com/radius-project/radius/pull/7950
+
+### Implement PostgreSQL datastore
+https://github.com/radius-project/radius/pull/7961
+
+## Breaking changes
+https://github.com/radius-project/radius/pull/7906
```

Comment:
@willtsai could you please help updating these. I have copied over PRs of a few items from changelog that might be good candidates for highlights.